src/generic_solver.cpp

```diff
@@ -495,8 +491,8 @@ Sort GenericSolver::make_sort(const DatatypeDecl & d) const
     // Exact functionality of make_genericsort, without the linking
     // errors (undefined reference when I call a new
     // make_generic_sort).
-    //Sort dt_sort = make_generic_sort(curr_dt);
-    Sort dt_sort = make_shared<GenericDatatypeSort>(curr_dt);
+    Sort dt_sort = make_generic_sort(curr_dt);
+    //Sort dt_sort = make_shared<GenericDatatypeSort>(curr_dt);
```

Comment:
...

Reply:
I am confused by this, because looking through my branch and the most recent push, this commented line is nowhere to be found.